gammapy/modeling/models/core.py

```diff
@@ -105,10 +105,20 @@ def copy(self):
         """A deep copy."""
         return copy.deepcopy(self)
 
-    def to_dict(self):
+    def to_dict(self, full_output=True):
         """Create dict for YAML serialisation"""
         tag = self.tag[0] if isinstance(self.tag, list) else self.tag
-        return {"type": tag, "parameters": self.parameters.to_dict()}
+        params = self.parameters.to_dict()
+
+        if full_output is False:
```

Comment:
Please just use `if not full_output:` here. The `is` pattern kind of works accidentally here...